src/frontends/pytorch/src/op/arange.cpp

```diff
@@ -108,6 +109,10 @@ OutputVector translate_arange_fx(const NodeContext& context) {
     if (context.has_attribute("dtype")) {
         dtype = context.get_attribute<element::Type>("dtype");
     }
+    if (end.get_partial_shape().rank().is_dynamic() ||
+        (end.get_partial_shape().rank().is_static() && end.get_partial_shape().rank().get_length() == 1)) {
+        end = context.mark_node(std::make_shared<ov::op::v0::Squeeze>(end, zero));
+    }
```

Comment:
I don't like this change. You only do this for `end`, while `start` and `step` are similar. Could you make a test? `torch.export` test can cover this case.

Reply:
We can do the same for start and step as well. But it seems like the unit test for arange already have an issue that torch dynamo removes this op. It never reaches to fx_openvino as a single op. Need to fix this issue before updating the test.

Reply:
Arange op with dynamic inputs does not seem to be fully  supported in torch dynamo. So, it is excluded from the graph and never gets into OV in the current unit tests. So, this seems to be an issue with the existing arange unit test now (could be observed in OV master as well). I didn't observe this issue for arange ops as part of Llama2. It seems like we need an actual fix for the current issue before adding a new test for the updates from this PR. Do we need this to be resolved in this PR?